Updated readme for review. (#6)
IBTTA-Blockchain-Working-Group/Lab-Group: README.md

# Lab-Group

### Introduction

> [IBTTA] Working Group for blockchain technologies has been formed to understand the impact of blockchain within transportation. This group is formed to bring together experts from various parts of tolling and transportation industry to help outline applicability of blockchain (distributed-ledger, to contrast from cryptocurrency) technology.

Reviewer: ```suggestion
The [IBTTA](https://www.ibtta.org) Blockchain Working Group for has been formed under the [Emerging Technologies Committee](https://my.ibtta.org/About-Us/Committees) to understand the impact of blockchain innovation within transportation. This group brings together experts from various disciplines in tolling and transportation to help outline the applicability of distributed ledgers, smart contracts, and tokenization.
```

### Interaction with Stakeholders

We love to interact with stakeholders so that we are building our mvps based on a real need. If you are interested in discussing a particular aspect of Tolling and Transportation that you would like to see addressed, please reach out to us, we will be happy to discuss and prioritize them with you.

### Definition of MVPs - 2021

> Our goal for the MVPs that we would like to develop is to cover the following aspects of Tolling and Transportation:

Reviewer: ```suggestion
Our goal for the MVPs that we would like to develop is to cover the following aspects of Tolling and Transportation:

```

    - Fleet management
    - Interoperability (IOP)

> We use [Monday](https://milliganpartners.monday.com/boards/1200830450/), a public company, to capture our work. Please reach out to (maintainer's name here) to be added to the board to influence our backlog and priorities.

Reviewer: ```suggestion
We use [Monday](https://milliganpartners.monday.com/boards/1200830450/), a public company, to capture our work. Please reach out to @kevinko102496 to be added to the board to influence our backlog and priorities.
```

### Definition of Smart Contracts

A sample contract for [Toll Guarantee State](./docs/TollGuaranteeStateIBTTA.pdf) state machine that is relevant for agencies supporting the IAG format. Agencies supporting NIOP don't have to incur this risk attribution. We use this contract as an opportunity to propose continuous compounding for managing risk on a blockchain ledger (or any distributed system) where the time intervals used in traditional finance as various parties may be spread across various timezones.

### Choice of Ledgers
Our choice of ledger is based on some of the features listed here:
  - Sharding is built-in
  - Eco-friendly mining
  - Safe-by-design smart contract language (more about this later)

We began the effort by evaluating [Zilliqa](https://www.zilliqa.com), though we will be looking to port to other languages/ledgers if a suitable ledger emerges.

### How to Participate in the Group

> Though we use the term "lab" we consider us to be a bunch of like-minded folks who like to maintain a collegial environment to exchange and bounce off ideas/code etc. If you see anything you like or don't like, please send us a note and we will be happy to discuss them and perhaps create a user story or two to memorialize them.

Reviewer: ```suggestion
Though we use the term "lab," we consider ourselves to be a bunch of like-minded folks who like to maintain a collegial environment to exchange and bounce off ideas/code, etc. If you see anything you like or don't like, please send us a note and we will be happy to discuss them and perhaps create a user story or two to memorialize them.
```

Author: @kevinko102496 i am re-adding this section for now, please feel free to edit them or link to your changes about community. Thanks.

> ### Team/Acknowledgments

Reviewer: ```suggestion
### Team/Acknowledgments
We wish to thank these invaluable contributors and teammates:

* Dinkar Ganti, [Conduent](https://www.conduent.com)
* Kevin Ko, [Milligan Partners](https://www.milliganpartners.com)
* Matt Milligan, [Milligan Partners](https://www.milliganpartners.com)
* Devang Patel, [Kyra Solutions](https://www.kyrasolutions.com)

```
